### DISCUSSION TITLE
Dev Call - 30 Apr 2023

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#2512] [General]
Let's add agenda items and summarize the decisions made during the Dev Call here.

* VS Code extension: although the project's prql compiler seems to have been updated to 0.8.0, the actual extension on the Microsoft Marketplace hasn't been updated since March. What is necessary to trigger a rebuild/push to make it available for automatic update in VS Code?

* It looks as if the 0.8.1 release has broken the Playground. The SQL pane remains blank, and there are Javascript errors in the console. Do you need an issue submitted?

* Thanks, all, for committing and backporting lots of small fixes.

@max-sixty I cannot attend this morning, so I'll throw these over to the agenda. Thanks.

**Comment by max-sixty** [comment]
There was no dev call today — it's the first and third Sunday of the month!

But:
- those are helpful issues — definitely generally put them in separately. I'll take a look at these now.
- this sort of issue is a good idea to plan what we should discuss! I've been using the "Discussion" and "Priority" labels. Generally we should try and focus on changes and important issues, not just things that haven't made the issue tracker yet

**Comment by max-sixty** [comment]
I'm updated the vscode version — someone just needs to bump the version like this: https://github.com/PRQL/prql-vscode/pull/192 — PRs welcome in the future! 

And then to do a release like https://github.com/PRQL/prql-vscode/releases/tag/0.8.1

I opened an issue for the playground, I'm still having a look
